pkg/controllers/state/informer/provisioner.go

```diff
@@ -32,7 +32,7 @@ import (
 
 var _ corecontroller.TypedController[*v1alpha5.Provisioner] = (*ProvisionerController)(nil)
 
-// ProvisionerController reconciles provisioners to re-trigger consolidation on change.
+// NodePoolController reconciles provisioners to re-trigger consolidation on change.
```

Comment:
find replace mistake?
```suggestion
// ProvisionerController reconciles provisioners to re-trigger consolidation on change.
```

Reply:
Good eye